### DISCUSSION TITLE
Cannot get Canyon running

### DISCUSSION THREAD

**Discussion by AnalogStick** [itzg/docker-minecraft-server#2070] [Q&A]
I've been trying to run canyon with minecraft beta 1.7.3 and I've been unsuccessful. I'm not sure what I'm doing wrong but here's the log.

[init] Running as uid=1000 gid=1000 with /data as 'drwxrwxr-x 2 1000 1000 4096 Apr  7 01:29 /data'
[init] Resolved version given b1.7.3 into b1.7.3 and major version b1.7
[init] Resolving type given CANYON
curl: (22) The requested URL returned error: 410

Any help would be appreciated.

**Comment by itzg** [comment]
Seems to be gone: 🤷‍♂️

![image](https://user-images.githubusercontent.com/988985/230634040-7b3f136d-00bb-4bc9-954d-7934a0a1abde.png)
